dev-util/rpi-imager/rpi-imager-1.8.5.ebuild

```
@@ -0,0 +1,99 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg
+
+DESCRIPTION="Raspberry Pi Imaging Utility"
+HOMEPAGE="
+https://www.raspberrypi.com/software/
+https://github.com/raspberrypi/rpi-imager
+"
+SRC_URI="https://github.com/raspberrypi/rpi-imager/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P}/src"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+
+KEYWORDS="~amd64"
+IUSE="gnutls qt5 qt6 zlib lzma zstd brotli bzip2 xz dbus gnutls openssl -telemetry"
```

```suggestion
IUSE="gnutls qt5 qt6 zlib lzma zstd brotli bzip2 xz dbus gnutls openssl telemetry"
```
Do not add -. It doesn't do anything.

---

I was reading this
>  Add + or - before the name of the use flag in IUSE to turn it on or off by default. 
https://devmanual.gentoo.org/general-concepts/use-flags/#iuse-defaults
but I guess its - by default.

---

Yes. Noticed that it says "Important: Adding - before a flag in IUSE is pretty much useless."

---

Yes. I guess I saw that but the point didn't come across very clearly.